source/FlagPole.hx

```diff
@@ -0,0 +1,83 @@
+package;
+
+import flixel.FlxBasic;
+import flixel.FlxG;
+import flixel.FlxObject;
+import flixel.FlxSprite;
+import flixel.group.FlxGroup;
+import flixel.math.FlxPoint;
+import flixel.system.FlxAssets.FlxGraphicAsset;
+import flixel.system.FlxSound;
+import flixel.util.FlxColor;
+import flixel.input.keyboard.FlxKey;
+import flixel.util.FlxPath;
+
+/**
+ * ...
+ * @author Cameron Yuan
+ */
+class FlagPole extends FlxSprite 
+{
+	
+	public var totalFlxGrp:FlxGroup;
```

Comment:
Regarding this, lines 50-52, and PlayState line 30:  Is it really worthwhile to put the pole and flag into a group like this?  Since you only ever check collisions on the pole itself and the group is only used during adding them to the PlayState, it seems like using 4 extra lines of code and some extra memory to avoid writing 1 extra line (adding the flag separately from the pole) in PlayState.